src/DocumentDB.ChangeFeedProcessor/PartitionManagement/ILeaseEx.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  Licensed under the MIT license.
+// ----------------------------------------------------------------
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement
+{
+    /// <summary>
+    /// Provides extensibility for <see cref="ILease"/> interface.
+    /// </summary>
+    public interface ILeaseEx : ILease
```

Comment:
Can we have this as separate interface rather than extending (and DocumentServiceLease implementing both)? Especially that we are going to deprecate it in v3 and forget about it. @kadukf  what do you think?